POCServer/src/java/DAL/UserHelper.java

```diff
@@ -0,0 +1,139 @@
+package DAL;
+
+import Model.Roles;
+import Model.User;
+import Model.UserRoles;
+import Util.HibernateUtil;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.hibernate.Query;
+import org.hibernate.Session;
+
+public class UserHelper {
+    protected Session session = null;
+
+    public UserHelper() {
+        if(session == null){
+            this.session = HibernateUtil.getSessionFactory().getCurrentSession();
+        }        
+    }
+    
+    public UserHelper(Session session) {
+        this.session = session;
+    }
+    
+    public List<User> getUsers() {
+        List<User> users = new ArrayList();
+        org.hibernate.Transaction tx = session.beginTransaction();
```

Comment:
please could you use 'transaction' instead of 'tx', to improve readability of the code